index.js

```diff
@@ -0,0 +1,19 @@
+const toggleBtn = document.querySelector('#toggleBtn');
+const ulElem = document.querySelector('#menuList');
+
+toggleBtn.addEventListener('click', () => {
```

Comment:
- [ ] great job so far with the changes. please like the previous reviewer noted, kindly add the correct background image like that of figma.
 - [ optional] you can align your close icon with the other link icons
your design 
![Screenshot (424)](https://user-images.githubusercontent.com/65298048/188518217-89eeee91-39f6-43ae-aebd-f764b000e359.png)
figma design
![Screenshot (424)_LI](https://user-images.githubusercontent.com/65298048/188518229-482ca6d7-5eb3-4cfe-84f5-1a81ee1fd92a.jpg)